lrtable/src/lib/statetable.rs

```diff
@@ -554,7 +554,7 @@ fn resolve_shift_reduce<StorageT: 'static + Hash + PrimInt + Unsigned>(
     pidx: PIdx<StorageT>,
     stidx: StIdx, // State we want to shift to
     shift_reduce: &mut Vec<(TIdx<StorageT>, PIdx<StorageT>, StIdx)>,
-    state: StIdx // State in which the conflict occured
+    shift_stidx: StIdx // State in which the conflict occured
```

Comment:
This variable name seems odd, given the comment on `stidx`. Can we come up with something better?

Reply:
I was just following your suggestion, though I do agree this can be somewhat confusing. I guess your other suggestion `conflict_stidx` it is then.